src/parser/csql_grammar.y

```diff
@@ -27533,3 +27568,48 @@ pt_jt_append_column_or_nested_node (PT_NODE * jt_node, PT_NODE * jt_col_or_neste
     }
 }
 
+static PT_NODE *
+pt_make_paren_expr_list (PT_NODE * exp)
+{
+
+  PT_NODE *val, *tmp;
+
+  bool is_single_expression = true;
+  if (exp && exp->next != NULL)
+    {
+      is_single_expression = false;
+    }
+
+  if (is_single_expression)
```

Comment:
These lines are equal as below.
`if (exp && expr->next == NULL)`
the is_single_expression is unnecessary. And no need to check exp is not null from the next line by this change.

Reply:
I agree. I fixed it